addons/zotero/migrations/0005_auto_20180216_0849.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: we generally give our custom migrations descriptive names, e.g. `0005_zotero_personal_libraries_2018....py`